sdk/identity/azure-identity/src/main/java/com/azure/identity/Util.java

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.identity;
+
+import com.azure.core.credential.TokenCredential;
+import com.azure.core.http.HttpHeaderName;
+import com.azure.core.http.HttpMethod;
+import com.azure.core.http.HttpPipeline;
+import com.azure.core.http.HttpPipelineBuilder;
+import com.azure.core.http.HttpRequest;
+import com.azure.core.http.HttpResponse;
+import com.azure.core.http.policy.BearerTokenAuthenticationPolicy;
+import com.azure.core.util.Context;
+
+import java.util.function.Supplier;
+
+public class Util {
```

Comment:
Does this need to be public API? I only see this being used as a sample. And if we're making this public API I think we should give this class a better name than Util, maybe BearerTokenUtils?

Reply:
Was discussing this with @srnagar offline; going with `AuthenticationUtil`. We expect this is used by folks who are using the OpenAI libraries and need tokens.